exec/vector/src/test/java/org/apache/drill/exec/vector/VariableLengthVectorTest.java

```diff
@@ -37,6 +37,7 @@ public void testSettingSameValueCount()
   {
     try (RootAllocator allocator = new RootAllocator(10_000_000)) {
       final MaterializedField field = MaterializedField.create("stringCol", Types.required(TypeProtos.MinorType.VARCHAR));
+      @SuppressWarnings("resource")
```

Comment:
@paul-rogers Thank you for making the changes. Can you please double check whether it is necessary to suppress the warning here. It is confusing why the same warning suppression is removed in some places and added in others.

Reply:
These warnings may be unique to Eclipse. In this case, the vector is created, but not used in try-with-catch block which is the key purpose of anything that derives from `AutoCloseable`. Drill uses that class simply as a "close() without throwing exceptions" interface.

Since the warning really does not help in Drill, (and may not appear in IntelliJ), I can just turn the warning off in Eclipse so I'm not tempted to silence the warnings by inserting these annotations.